sdk/core/ts-http-runtime/scripts/azure-diff.ts

```diff
@@ -0,0 +1,182 @@
+import { parseArgs } from "node:util";
+import fs from "node:fs/promises";
+import path from "node:path";
+import { spawn } from "node:child_process";
+import { exit } from "node:process";
+
+/**
+ * Mapping of files in the unbranded package to their Azure counterparts
+ *
+ * For each source file in the unbranded package, we will loop through these entries and find the first matching key. A key is considered
+ * a match if the unbranded file's path starts with the key.
+ *
+ * For example, any file under src/logger will map to ../logger/src (the Azure logger package), except for src/logger/logger.ts, which will get mapped
+ * to ../logger/src/index.ts, since that entry comes before src/logger in the object.
+ */
+const UNBRANDED_TO_AZURE_MAPPINGS = {
+  // Logger index file was renamed
+  "src/logger/logger.ts": "../logger/src/index.ts",
+
+  "src/logger": "../logger/src",
+  "src/auth": "../core-auth/src",
+  "src/abort-controller": "../abort-controller/src",
+  "src/client": "../core-client-rest/src",
+  "src/policies": "../core-rest-pipeline/src/policies",
+  "src/retryStrategies": "../core-rest-pipeline/src/retryStrategies",
+  "src/tracing": "../core-tracing/src",
+  "src/util": "../core-util/src",
+
+  src: "../core-rest-pipeline/src",
+};
+
+const {
+  values: { update = false, "output-path": outputPath = "./review/azure-core-comparison.diff" },
+} = parseArgs({
+  options: {
+    update: {
+      type: "boolean",
+    },
+    "output-path": {
+      type: "string",
+      short: "o",
+    },
+  },
+});
+
+/**
+ * Recursively find all files in the directory
+ * @param dir - Directory to find files in
+ * @returns - an array containing paths to all files in the directory
+ */
+async function find(dir: string): Promise<string[]> {
```

Comment:
Up to you, but I think you already get [glob](https://www.npmjs.com/package/glob) transitively so adding it as a devDep and using it won't be harmful and can simplify this a bunch (says the same guy who mentioned appreciating that you used node APIs above - the hypocrisy is not lost on me)

Reply:
Thanks, but don't need either this helper or glob any more with the change in approach